### ISSUE TITLE
still working?

### ISSUE THREAD

**Issue by dcsan** [yonggao/meteor-wechat-robot#1]
hey, thanks for packaging this, but i notice its pretty old. is it still working?

if so, what is the recommended way to get started?

**Comment by yonggao** [comment]
No, not any more. I suggest you take the node module here as starting point to integrate with wechat - https://github.com/node-webot/wechat
